test/index.js

```diff
@@ -464,6 +464,64 @@ describe('def', function() {
                    'as its first argument; received 0.5'));
   });
 
+  it('supports enumerated types', function() {
+    //  TimeUnit :: Type
+    var TimeUnit = $.EnumType(['milliseconds', 'seconds', 'minutes', 'hours']);
+
+    var def = $.create($.env.concat([TimeUnit, $.ValidDate, $.ValidNumber]));
+
+    //  convertTo :: TimeUnit -> ValidDate -> ValidNumber
+    var convertTo =
+    def('convertTo',
+        {},
+        [TimeUnit, $.ValidDate, $.ValidNumber],
+        function recur(unit, date) {
+          switch (unit) {
+            case 'milliseconds': return date.valueOf();
+            case 'seconds':      return recur('milliseconds', date) / 1000;
+            case 'minutes':      return recur('seconds', date) / 60;
+            case 'hours':        return recur('minutes', date) / 60;
+          }
+        });
+
+    throws(function() { convertTo('days', new Date(0)); },
+           errorEq(TypeError,
+                   '‘convertTo’ expected a value of type ' +
+                   '("milliseconds" | "seconds" | "minutes" | "hours") as ' +
+                   'its first argument; received "days"'));
```

Comment:
I feel bad even requesting this change, as it's the nittiest of nits. I like to wrap these messages after the type representation, if possible. In this case:

```diff
                    '‘convertTo’ expected a value of type ' +
-                   '("milliseconds" | "seconds" | "minutes" | "hours") as ' +
-                   'its first argument; received "days"'));
+                   '("milliseconds" | "seconds" | "minutes" | "hours") ' +
+                   'as its first argument; received "days"'));
```

Thank you for accommodating my fastidiousness!